Makefile

```diff
@@ -63,7 +63,8 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+	$(YARN) check
```

Comment:
The lock file gets generated when you run the `yarn` command, that's what I meant by renaming `setup` to `yarn.lock`, that's when the file is either generated or updated.